package.json

```diff
@@ -52,28 +52,30 @@
   },
   "homepage": "https://carbon.sage.com",
   "peerDependencies": {
-    "@sage/design-tokens": "^4.17.0",
+    "@sage/design-tokens": "^4.31.0",
```

Comment:
comment: good idea to update this, let's update the [guidance](https://github.com/Sage/carbon/blob/36093a5b04f82ed9eafedbfd40d07e6f0572aa9e/docs/installation.stories.mdx#L26) to reflect it as well.

Reply:
Good point, though will consumers get the latest minor version of `@sage/design-tokens` anyway?

Reply:
They will but we may as well update it as we've had people point it out recently